model/vcs/registry.yaml

```diff
@@ -261,8 +261,8 @@ groups:
               value: gitlab
               brief: "[GitLab](https://gitlab.com)"
               stability: development
-            - id: gittea
-              value: gittea
+            - id: gitea
```

Comment:
```suggestion
            - id: gittea
              value: gittea
              brief: "Deprecated, use `gitea` instead"
              deprecated: "Replaced by `gitea`."
              stability: development
            - id: gitea
```

(it will fix CI)

Reply:
Thanks!